resources/integration_test/testdata/integration.yml

```diff
@@ -0,0 +1,20 @@
+dataset:
```

Comment:
Either as part of this ticket or a follow-up ticket (possibly SQL-563 which I have in progress), we should also expand the data restoration code to support creation of views. It should be pretty straightforward, including a `view` key to name a view where the `collection` is the source. Instead of `docs` we'll have `pipeline`. Let me know if you want to do that as part of this ticket or if you'd prefer to spin this work off to a follow-up ticket!

Reply:
That would be great to have.  Let's do this in a follow-up ticket.  Feel free to pick it up as part of SQL-563 if you would like.

Reply:
Ok cool. I'll plan on including that on 563 for now; I'll get on it after this is merged. When I get to it, I may still decided to bump it to a follow-up ticket, but we'll cross that bridge when we get there.